### ISSUE TITLE
히토미 다운로더의 픽시브 계정 인식이 되지 않습니다.

### ISSUE THREAD

**Issue by Act-20** [KurtBestor/Hitomi-Downloader#2729]
분명 정확한 아이디와 비밀번호를 Pixiv 설정의 로그인란에
적었는데도 스크린샷과 같은 에러가 발생합니다.

**Comment by dltkdgus03316** [comment]
저도 마찬가지입니다

**Comment by godknowswho** [comment]
저도 마찬가지입니다. 13일까지만해도 멀쩡히 됬는데 갑자기 안되네요.

**Comment by god111dog** [comment]
혹시 장시간 비밀번호를 바꾸지 않아서 픽시브에 처음 접속할 때 비밀번호를 변경하라고 하지는 않나요? 저도 같은 증상이었는데 혹시나 해서 비밀번호를 변경했더니 다시 작동되고 있습니다.

**Comment by STRESSSSS** [comment]
저도 안되네요 ㅜ

**Comment by KurtBestor** [comment]
웹브라우저에선 로그인 되나요?

**Comment by STRESSSSS** [comment]
> 웹브라우저에선 로그인 되나요?

네 픽시브 웹브라우저에선 잘 됩니다. 방금 확인해보니 웹브라우저에서 로그인할때 비번을 바꾸라는 권장 메시지가 뜨는데 그때 비번으로는 로그인이 안되다가 비번을 변경후에는 로그인이 잘되네요. 아무래도 관계가 있지 않나 생각합니다. 일단 비번 바꾸고나서는 잘 됩니다.

**Comment by Act-20** [comment]
> 웹브라우저에선 로그인 되나요?

저도 웹브라우저에서는 잘 됩니다.

**Comment by wjsqhr** [comment]
픽시브 로그인 원래 이메일로 하는거 맞죠? 비번을 바꿔도 안되던데
